### ISSUE TITLE
[Bug]: Accessibility: Empty <ul> tag .jp-carousel-image-exif

### ISSUE THREAD

**Issue by masperber** [Automattic/jetpack#38839]
### Quick summary

https://www.boia.org/ report:

> List container <UL> Tag should only contain proper child items
>
> The tag `<ul class="jp-carousel-image-exif" style="display: none;"></ul>` lacks any proper child items.

### Steps to reproduce

 1. Create any site with Jetpack Carousel Image module active (true on wpcom by default)
 2. Add a new post
 3. Add a gallery block
 4. Add several images, at least one of which doesn't have exif data
 5. Save the post
 6. View the front-end, select an image without exif data

### What you expected to happen

Search the HTML for `jp-carousel-image-exif`, the ul will be empty

Since this tag is used for presentation, a non-list tag should be used.

### What actually happened

An empty <ul> tag is used.

### Context

WordPress.com customer report

### Platform (Simple, Atomic, or both?)

Simple, Atomic

### Reproducibility

Consistent

### Severity

All

### Available workarounds?

No but the platform is still usable

**Comment by dsas** [comment]
Moved to Jetpack, the [empty exif ul](https://github.com/Automattic/jetpack/blob/a8217ebc467e67382983881d243ee7313b0f366b/projects/plugins/jetpack/modules/carousel/jetpack-carousel.php#L766) comes from Jetpack's carousel module rather than any particular theme.

**Comment by dsas** [comment]
@masperber can you shed some further light on this? The message given is a bit confusing and valid content for a `ul` element according to [spec](https://html.spec.whatwg.org/#the-ul-element) is "Zero or more li and script-supporting elements."

**Comment by masperber** [comment]
Hi @dsas, thanks for following up on this.

This report was opened nearly 2 years ago, and unfortunately it seems that I didn't notate the full context of the report. Based on my best recollection, I believe that a WordPress.com customer reached out and was concerned by an accessibility report that they generated by running their site through the tool found at https://www.boia.org/

While my understanding is that it is semantically valid to have an empty `ul` element, there are accessibility concerns about this pattern. [Here's some information that I was able to find](https://help.siteimprove.com/support/solutions/articles/80001051793-accessibility-rule-container-element-is-empty-explained).

**Comment by dsas** [comment]
Cheers, boia.org didn't show me the same error but given the link you shared, perhaps the specs aren't exactly clear on this point! Regardless, a UL that is then later updated by JS might need to use [`aria-busy`](https://www.w3.org/TR/wai-aria-1.1/#aria-busy) or something to become properly accessible.

**Comment by ryelle** [comment]
I don't think the empty ul is an accessibility issue, at least not in a "compliance with the guidelines" kind of way. It would probably be more clear if the `ul` had a heading or other accessible title, but an empty ul should be fine.

I would also avoid adding aria here, this isn't a live region so that attribute wouldn't apply, and the entire element should be hidden when it would be "busy". Once it's available, it should be filled in (or intentionally empty, if no exif data).
